packages/ui-components/src/helpers/common/validation.ts

```diff
@@ -77,3 +77,17 @@ export const stringContains =
       return match.test(val) ? true : message
     }
   }
+
+export const isUrl: GenericValidateFunction<string> = (value) => {
+  if (VALID_HTTP_URL.test(value)) {
+    return true
+  }
+  return 'Invalid URL'
+}
+
+export const isItemSelected: GenericValidateFunction<[]> = (val) => {
```

Comment:
I'm confused why TS doesn't throw an error for `[]` as a type, because it's not clear what kind of array is it supposed to be. Since you explicitly don't care about the type of the individual array items you can just do `unknown[]`

https://dmitripavlutin.com/typescript-unknown-vs-any/